k2/csrc/cuda_headers.cuh

```diff
@@ -0,0 +1,31 @@
+/**
+ * @brief
+ * Put cuda runtime headers here.
+ * With this file be included, code static analysis works.
+ *
+ * @copyright
+ * Copyright (c)  2020  Xiaomi Corporation (authors: Daniel Povey
+ *                                                   Haowen Qiu)
```

Comment:
seems this file is created by you, so there should be no Dan's name and mine.

Reply:
Corrected. And FYI this file would be deleted when I found it's useless for us. Let's review it in new PRs. As this PR wouldn't be merged, dan said one PR should only solve one issue. Thx review here would keep these changes in new PRs.